### PR TITLE
Get button text color from theme

### PR DESCRIPTION
**Pull request by possibilities**
Would be nice to have the button text color come from the theme. Unsure this is precisely how we'd go about it but should work. What do you think?

### CODE REVIEW

File: src/LogMonitorButton.js [hunk 1]
```diff
@@ -66,6 +66,7 @@ export default class LogMonitorButton extends React.Component {
   render() {
     let style = {
       ...styles.base,
+      color: this.props.theme.base06,
```
Comment (possibilities) [inline src/LogMonitorButton.js]: Might make sense to drop color from the base styles above too.

**Comment by gaearon** [comment]
Can you show before and after screenshots with different themes?

**Comment by gaearon** [comment]
Ping @possibilities, any more info on this?

**Comment by possibilities** [comment]
Ah, lost interest in this. If that changes I'll open a new PR that doesn't
suck (:

**Comment by gaearon** [comment]
:rainbow:

**Pull request closed without merging**